ios/Podfile

```diff
@@ -38,18 +38,17 @@ target 'pillarwallet' do
   pod 'BVLinearGradient', :path => '../node_modules/react-native-linear-gradient'
   pod 'Firebase/Core', '~> 5.0.0'
   pod 'Firebase/Database'
-  pod 'Firebase/Analytics'
   pod 'Firebase/Auth'
   pod 'Firebase/Firestore'
   pod 'Firebase/Messaging'
   pod 'Firebase/DynamicLinks'
-  pod 'Firebase/RemoteConfig'
   pod 'Firebase/Storage'
   pod 'Firebase/Performance'
   pod 'ReactNativePermissions', :path => '../node_modules/react-native-permissions'
   pod 'react-native-fetch-blob', :path => '../node_modules/react-native-fetch-blob'
   pod 'RNKeychain', :path => '../node_modules/react-native-keychain'
 
+  pod 'Intercom', '~> 5.2.1'
```

Comment:
This locks Intercom Pod version to the one that matches it's react native library javascript dependancy.